package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "browserify-css": "^0.8.2",
     "debug": "^2.2.0",
+    "deep-assign": "^2.0.0",
```

Comment:
if we're using this, want to remove the `object-assign` dependency and all its usage?